core/block_rendering_rewrite/block_render_draw.js

```diff
@@ -288,7 +288,8 @@ Blockly.BlockRendering.Drawer.prototype.drawInternals_ = function() {
 Blockly.BlockRendering.Drawer.prototype.dealWithJackassFields_ = function(field) {
```

Comment:
Wasn't there suppose to be a TODO to remove this method before it goes into master?